app/models/school_device_allocation.rb

```diff
@@ -116,10 +116,4 @@ def cap_implied_by_order_state(order_state:, given_cap: nil)
   def has_virtual_cap_feature_flags?
     school&.responsible_body&.has_virtual_cap_feature_flags? || false
   end
-
-  def cap_lte_allocation
```

Comment:
I saw this as well - we're not using it at all and it's a duplicate of the validator.